Ghidra/Features/Decompiler/src/decompile/cpp/sleigh.cc

```diff
@@ -106,7 +58,7 @@ void PcodeCacher::addLabel(uint4 id)
 void PcodeCacher::clear(void)
 
 {
-  curpool = poolstart;
+  pool.clear();
```

Comment:
`std::vector::clear` doesn't require a change in the capacity and whether or not it does so is implementation defined. I would expect it to only change the size and if you wanted the capacity to be changed one would use `std::vector::shrink_to_fit`.